src/enforcer/HolographERC721.sol

```diff
@@ -321,7 +324,30 @@ contract HolographERC721 is Admin, Owner, HolographERC721Interface, Initializabl
     require(_isApproved(sender, tokenId), "ERC721: sender not approved");
     require(from == _tokenOwner[tokenId], "ERC721: from is not owner");
     if (_isEventRegistered(HolographERC721Event.bridgeOut)) {
-      data = SourceERC721().bridgeOut(toChain, from, to, tokenId);
+      bytes memory sourcePayload = abi.encodeWithSelector(
```

Comment:
@ACC01ADE can we make a comment about this? It looks like this is going to call the bridgeOut function but its not obvious.

Reply:
Added commentary to this on ERC721 and ERC20. Also provided commentary on `_sourceCall`.